core/deploy_event.py

```diff
@@ -0,0 +1,69 @@
+from core.models import Event
+
+
+def copy_event(previous_event, event_date):
+    """
+        Copy event from the previous one. This does the following steps:
+        - change name of previous event to {name} #{number}, i.e.
+          Django Girls London #1
+        - create new event based on previous event
+        - copy content and menu from previous event
+    """
+    number = Event.objects.filter(city=previous_event.city).count()
```

Comment:
should filter by country as well, there could be two cities of the same name in different countries